### PR TITLE
change: [M3-8353] - Update `xterm` and make LISH fit viewport

### PR DESCRIPTION
**Pull request by bnussman-akamai**
## Description 📝

- Updates `xterm` to `@xterm/xterm` (see [here](https://github.com/xtermjs/xterm.js/releases/tag/5.4.0)) 📦 
- Updates both Lish and Glish to fit the viewport 🖥️ 
- Updates Lish to open in new tab rather than a new window 🪟 
- Remove old / unused CSS 🗑️ 
- Removes code that manually sets the browsers viewport size 🗑️ 

### Glish

| Before  | After   |
| ------- | ------- |
| <video src="https://github.com/user-attachments/assets/f3b61611-1c52-4bec-aa00-d24cc1985dfd" /> | <video src="https://github.com/user-attachments/assets/ad1959ab-17bf-4ba4-a1c9-1afd33b955d0" /> |

### Lish

| Before  | After   |
| ------- | ------- |
| <video src="https://github.com/user-attachments/assets/9cd77295-46bb-41f4-95ee-ca9df88307c5" /> | <video src="https://github.com/user-attachments/assets/9774e7c8-b9ea-4a33-a5e9-ee577648d084" /> |

## How to test 🧪

- Test Lish and Glish
  - Verify general styles / CSS looks good 🎨 
  - Verify Lish and Glish still work and connect as expected 🔌 
  - Verify Lish and Glish properly scale and fix the viewport of the window 🖥️ 

## As an Author I have considered 🤔

- [ ] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/src/features/Lish/Weblish.tsx [hunk 1]
```diff
@@ -113,16 +119,26 @@ export class Weblish extends React.Component<CombinedProps, State> {
...
     this.terminal.open(terminalDiv as HTMLElement);
 
+    window.onresize = () => {
+      this.fitAddon.fit();
+    };
+
+    setInterval(() => {
+      this.fitAddon.fit();
+    }, 2000);
+
```
Comment (mjac0bs) [inline packages/manager/src/features/Lish/Weblish.tsx:141]: Cool, this simplifies things a lot!

File: packages/manager/src/features/Lish/Glish.tsx [hunk 2]
```diff
@@ -1,31 +1,15 @@
...
-const useStyles = makeStyles()(() => ({
-  container: {
-    '& canvas': {
-      display: 'block',
-      margin: 'auto',
-    },
-  },
-  errorState: {
-    '& *': {
-      color: '#f4f4f4 !important',
```
Comment (mjac0bs) [inline packages/manager/src/features/Lish/Glish.tsx:24]: I think we lost the white text we'll need on the black background.

This branch:
Screenshot 2024-07-18 at 2 37 46 PM

Prod:
Screenshot 2024-07-18 at 2 38 08 PM

**Review by mjac0bs — APPROVED** [review]
Thanks for the changesets and all the clean up. 🧹  

Lish and Glish functionality seemed to be working without regressions, the window scaling works as expected, and styles LGTM other than the one regression I noticed with the error state black text.

**Comment by carrillo-erik** [comment]
The changes look good and phenomenal job with the CSS clean up. My only reservation is with the `!important` declarations introduced. I assume this choice was better than the alternative. Good job.

**Review by carrillo-erik — APPROVED** [review]

**Pull request merged**